.github/workflows/build.yml

```diff
@@ -37,22 +38,34 @@ jobs:
           echo "Flags:     ${{ steps.buildx.outputs.flags }}"
           echo "Platforms: ${{ steps.buildx.outputs.platforms }}"
 
-      - name: Build the image multiplatform
-        run: docker buildx build --output "type=image,push=false" --platform linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x  --tag mendhak/http-https-echo:latest --file ./Dockerfile .
+      - name: Docker metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            mendhak/http-https-echo
```

Comment:
I see this seems to be a Github Action way of what I was doing with the git tag here:

```
run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
```

Reply:
which was to get the git tag out and use that as the docker tag. OK

Reply:
Correct, this action does that automatically. The table under https://github.com/docker/metadata-action#basic gives a nice overview of the default values that it will generate